src/docs/design-systems/what-is-design-system.md

```diff
@@ -0,0 +1,30 @@
+---
+title: What is a Design System
+description: What is a Design System
+url: /docs/what-is-design-system
+contributors:
+  - dotNetkow
+---
+
+# What is a Design System?
+
+A Design System consists of UI components and a clearly defined visual style, released as both code implementations and design artifacts. When adopted by all product teams, a more cohesive customer experience emerges.
+
+There are several aspects that Design Systems consist of.
+
+## Components
+A component is a standalone UI element designed to be reusable across many projects. Its goal is to do one thing well while remaining abstract enough to allow for a variety of use cases. Developers can use them as building blocks to build new user experiences. A key benefit quickly becomes clear: not having to worry about the core design and functionality of each component every time they use them. Examples include buttons, links, forms, input fields, and modals.
+
+## Patterns
+A pattern is an opinionated use of components. Often, multiple components are combined together in order to create a standardized user experience (UX). Their key benefit is intertwined: UX as well as developer experience (DX). After implementing patterns, users will understand the application better and accomplish their tasks faster. When the development team understands the proper way to use components together, software applications become easier to use. Examples include saving data to the system, capturing data from forms, and filtering and analyzing data.

```

Comment:
@dotNetkow 
## What is "an opinionated use?"

I'm currently working on a [Japanese translation of this document](https://stencil-site-ja.firebaseapp.com/docs/faq), and I have thought that the word "opinionated" has a negative connotation to it, as in "an arrogant and opinionated man" (an example actually taken from the dictionary.) This, however, is obviously not what is meant here.

If you could clarify the intention or perhaps give me a paraphrase of the sentence in question, that would be an immense help.

Thanks beforehand!

Reply:
Thank you for the very easy-to-understand explanation!